rayon-core/src/scope/mod.rs

```diff
@@ -15,6 +15,8 @@ use unwind;
 #[cfg(test)]
 mod test;
 
+/// Represents a fork-join scope which can be used to spawn any number 
+/// of tasks. See [`Scope()`] for more information.
```

Comment:
This isn't quite right. For one thing, the name of the function is not capitalized, and you also need a link. Something like this is probably right:

    See [`scope()`][s] for more information.
    
    [s]: fn.join.html